images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
Ah I think the issue is the OS.

The official ruby containers are build on debian but we need CentOS